x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/mapper/Mapper.java

```diff
@@ -220,13 +220,13 @@ private PhysicalPlan mapBinary(BinaryPlan bp) {
         return MapperUtils.unsupported(bp);
     }
 
-    private PhysicalPlan mapMerge(Merge merge) {
+    private PhysicalPlan mapFork(Fork fork) {
         List<PhysicalPlan> physicalChildren = new ArrayList<>();
-        for (var child : merge.children()) {
+        for (var child : fork.children()) {
             var mappedChild = new FragmentExec(child);
             physicalChildren.add(mappedChild);
         }
-        return new MergeExec(merge.source(), physicalChildren, merge.output());
+        return new MergeExec(fork.source(), physicalChildren, fork.output());
```

Comment:
I have kept the name `MergeExec` because from an execution standpoint we are merging results from multiple sub plans.
When we add support for multiple forks (fork after fork) - we would likely need a `ForkExec` too.

Reply:
that make sense to me. ++